collection_requirements.rst

```diff
@@ -157,11 +168,17 @@ Branch protections MUST be enforced:
 CI Testing
 ===========
 
-At a minimum ``ansible-test sanity`` MUST be run from the `latest stable ansible-base branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. We suggest to *additionally* run ``ansible-test sanity`` from the ansible/ansible ``devel`` branch so that you find out about new linting requirements earlier.
+At a minimum ``ansible-test sanity`` MUST be run from the `latest stable ansible-base/ansible-core branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. We suggest to *additionally* run ``ansible-test sanity`` from the ansible/ansible ``devel`` branch so that you find out about new linting requirements earlier, as well as all version of ``ansible-base``/``ansible-core`` that the collection supports.
```

Comment:
```suggestion
At a minimum ``ansible-test sanity`` MUST be run from the `latest stable ansible-base/ansible-core branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. We suggest to *additionally* run ``ansible-test sanity`` from the ansible/ansible ``devel`` branch so that you find out about new linting requirements earlier, as well as all versions of ``ansible-base``/``ansible-core`` that the collection supports.
```